src/jquery.addressfield.js

```diff
@@ -65,6 +65,9 @@
         url: configs.json,
         async: configs.async,
         success: function (data) {
+          if($(configs.fields.country)[0].children.length === 0){
+            $.fn.addressfield.initCountries($.fn.addressfield.transform(data), configs.fields.country);
```

Comment:
May also want to do the `transform(data)` once and save the results in a variable, then pass a variable into this and the `binder` call below.
```javascript
var transformedData = $.fn.addressfield.transform(data);
$.fn.addressfield.initcountries(transFormedData, configs.fields.country)
```